apps/alert_processor/lib/model/query.ex

```diff
@@ -82,6 +82,20 @@ defmodule AlertProcessor.Model.Query do
         LEFT JOIN subscriptions s ON u.id = s.user_id
         GROUP BY u.communication_mode
         """
+      },
+      %__MODULE__{
+        label: "Subscription info for users with invalid emails",
+        query: """
+        SELECT
+          u.email AS email,
+          u.communication_mode AS communication_mode,
+          COUNT(DISTINCT s.id) AS total_subscriptions,
+          COUNT(DISTINCT CASE WHEN NOT s.paused THEN s.id END) as unpaused_subscriptions
+        FROM users u
+        INNER JOIN subscriptions s ON u.id = s.user_id
```

Comment:
We probably want this to include users with no subscriptions (since we want to see _all_ invalid emails), so this should be a `LEFT JOIN`.